WaysofWork.md

```diff
@@ -32,7 +32,10 @@ When created in another repository try to still apply these rules as much as pos
     - Backlog: if it has been agreed as necessary work
     - Planned: agreed and we know when to address it or by when it needs done 
   - Points (effort): assign a value effort from 2 to 9. A value of 1 typically means that the task is so short it would be better merged with another, a 10 means that it is probably best split in more than one issue (maybe it needs to be a story instead of an issue); if you really want to assign a 1 or 10 (cannot be merged or split) do so
```

Comment:
Do we consider this still necessary, or rather, is this something we would like to implement?

Reply:
I think this is only helpful if we plan to do some kinda analysis with it afterwards